frontend/packages/ceph-storage-plugin/src/constants/bucket-class.ts

```diff
@@ -0,0 +1,50 @@
+export const bucketClassNameRegex: RegExp = /^[a-z0-9]+[a-z0-9-.]+[a-z0-9]+$/;
+export enum BucketClassTypes {
+  STANDARD = 'Standard',
+  NAMESPACE = 'Namespace',
+}
+export const bucketClassTypeRadios = [
+  {
+    id: BucketClassTypes.STANDARD,
+    value: BucketClassTypes.STANDARD,
+    label: 'Standard',
+    description:
+      'Data will be ingested by Multi-cloud object gateway, dedeuped, compressed and encrypted. The encrypted chunks would be saved on the selected backing stores. Best used when the applications would always use the OpenShift Container Storage endpoints to access the data.',
+  },
+  {
+    id: BucketClassTypes.NAMESPACE,
+    value: BucketClassTypes.NAMESPACE,
+    label: 'Namespace',
+    description:
+      'Data will be stored as is(no dedup, compression, encryption) on the namespace stores. Namespace buckets allows for connecting to existing data and serving from them. Best used for existing data or when other applications (and native cloud services) need to access the data from outside the OpenShift Container Storage.',
```

Comment:
Don't you have to translate these strings?

Reply:
i missed it. Now translated